itests/fevm_test.go

```diff
@@ -868,3 +868,32 @@ func TestFEVMGetBlockDifficulty(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, len(ret), 32)
 }
+
+func TestFEVMErrorParsing(t *testing.T) {
+	ctx, cancel, client := kit.SetupFEVMTest(t)
+	defer cancel()
+
+	e := client.EVM()
+
+	_, contractAddr := e.DeployContractFromFilename(ctx, "contracts/Errors.hex")
+	contractAddrEth, err := ethtypes.EthAddressFromFilecoinAddress(contractAddr)
+	require.NoError(t, err)
+	for sig, expected := range map[string]string{
+		"failRevertEmpty()":  "none",
+		"failRevertReason()": "Error(my reason)",
+		"failAssert()":       "Assert()",
+		"failDivZero()":      "DivideByZero()",
+		"failCustom()":       "0x09caebf3",
```

Comment:
How do you get "0x09caebf3"?
maybe add a comment on the side to explain

Reply:
I'm now calculating it inline.